app/services/classification/base.rb

```diff
@@ -133,6 +137,18 @@ def path_elements(path = nil)
 
     private
 
+    def check_valid(key, value)
```

Comment:
`def validate_lookup_value` perhaps is a bit more meaningful right away, otherwise I have to think what are we checking validity for